python_modules/libraries/dagster-k8s/dagster_k8s/external_resource.py

```diff
@@ -196,6 +196,13 @@ def build_pod_body(
     if "containers" not in spec:
         spec["containers"] = [{}]
 
+    if "restart_policy" not in spec:
+        spec["restart_policy"] = "Never"
+    elif spec["restart_policy"] == "Always":
+        raise DagsterInvariantViolationError(
+            "A restart policy of Always is not allowed, computations are expected to complete."
+        )
```

Comment:
not sure why local kind was passing without this set correctly

Reply:
minor nit but this feels like a weird intermixing of error checking and business logic. Prefer to move invariant checking "upfront" generally.

```python
if spec.get("restart_policy") == "Always":
   ... # raise
   
if "restart_policy" not in spec:
    spec["restart_policy"] = "Never"
```

I assumed there was an `elif` there for more intricate reasons.

Completely minor nit and feel free to ignore.